quickcomm/urls.py

```diff
@@ -8,4 +8,7 @@
     path('register/', views.register, name='register'),
     path('create/plain', views.create_post, name='create_plain'),
     path('create/markdown', views.create_markdown, name='create_markdown'),
+    path('post/<uuid:post_id>/', views.post_view, name='post_view'),
+    path('post/<uuid:post_id>/post_liked', views.post_like, name='post_like'),
+    path('post/<uuid:post_id>/post_comment', views.post_comment, name='post_comment')
```

Comment:
this is probably fine for part 1, but once we need to join with other servers, we should follow the URLs in the spec. Which is to say we use posts/{POST_ID}/comments to post a comment (and the GET method on that same URL to view all comments on a given post).

Reply:
I actually spoke to @jaspreetkaursohal about this in person and okayed this, just for part 1. This URL is not meant to be used externally, only for our UI. In the future, the API will include a like, which will be client-side called via AJAX.